routes/genres.js

```diff
@@ -0,0 +1,53 @@
+const { Genre, validate} = require('../models/genre')
+const express = require('express');
+const router = express.Router();
+
```

Comment:
you can substitute the functions in the route handlers with imports from controller files.
ask riya and sreelekshmi for samples